lib/optimize/SplitChunksPlugin.js

```diff
@@ -145,23 +145,25 @@ const MinMaxSizeWarning = require("./MinMaxSizeWarning");
  * @property {Record<string, number>} sizes
  * @property {Set<Chunk>} chunks
  * @property {Set<Chunk>} reuseableChunks
- * @property {Set<string>} chunksKeys
+ * @property {Set<bigint>} chunksKeys
  */
 
 const defaultGetName = /** @type {GetName} */ (() => {});
 
 const deterministicGroupingForModules = /** @type {function(DeterministicGroupingOptionsForModule): DeterministicGroupingGroupedItemsForModule[]} */ (deterministicGrouping);
 
+/** @type {WeakMap<Module, string>} */
+const getKeyCache = new WeakMap();
+
 /**
  * @param {string} name a filename to hash
  * @returns {string} hashed filename
  */
 const hashFilename = name => {
-	return crypto
-		.createHash("md4")
+	const digest = /** @type {string} */ (createHash("md4")
```

Comment:
What about to use https://webpack.js.org/configuration/output/#outputhashfunction instead hard code `md4`

Reply:
done